sopel/module.py

```diff
@@ -115,6 +115,23 @@ def add_attribute(function):
     return add_attribute
 
 
+def echo(value):
```

Comment:
I agree [with @embolalia](https://github.com/sopel-irc/sopel/pull/1072#issuecomment-258612306):

> I don't think the `@echo` decorator should take a value. The default should be definitely not to get echoes, meaning the decorator only needs to mean "yup this thing can get echo messages".

This decorator should just set the `echo` function attribute to `True` if called. That's all it needs to do.